docs/jupyter_lite_config.json

```diff
@@ -0,0 +1,11 @@
+
+{
+    "LiteBuildConfig": {
+      "federated_extensions": [
+        "../jupyterlab_plugin_playground/labextension"
+      ],
+      "piplite_urls": [
+        "../dist"
```

Comment:
what benefit does the wheel actually provide at runtime in a kernel? do piplite things even work in xeus?